Makefile.dep

```diff
@@ -573,3 +573,8 @@ USEPKG := $(sort $(USEPKG))
 ifneq ($(OLD_USEMODULE) $(OLD_USEPKG),$(USEMODULE) $(USEPKG))
     include $(RIOTBASE)/Makefile.dep
 endif
+
+USEPKG_BUILDFIRST := nordic_softdevice_ble
```

Comment:
Yup. I consider this a quickfix for the problem at hand. I'd prefer full dependencies, or split download/patch and compile steps. But they'd be a lot more work to implement...